src/Graphics-Files/LzwGifDecoder.class.st

```diff
@@ -121,13 +121,13 @@ LzwGifDecoder >> handleCode: anInteger withPreviousCode: prevInteger on: aWriteS
 				the prefix index values and take the suffix each
 				time"
 				ifFalse: [ 
-					searchStack := Stack new.
+					searchStack := NewStack new.
```

Comment:
I have not checked these other parts of the system. I presume their tests are green after your rewrites.

Reply:
They are. :-)